### PR TITLE
[no-ticket] Workshop fixes

### PR DESCRIPTION
**Pull request by apoclyps**
### What's Changed

Adds fixes following a run through of the setup with Art.

### CODE REVIEW

File: services/events/project/scripts/README.md [hunk 1]
```diff
@@ -12,3 +12,12 @@ from project.scripts import meetup_tool as tool
 
 tool.create_meetups()
 ```
+
+### Running a one off tasks
```
Comment (conbon) [inline services/events/project/scripts/README.md]: "Running one off tasks"

File: services/events/project/scripts/README.md [hunk 2]
```diff
@@ -12,3 +12,12 @@ from project.scripts import meetup_tool as tool
...
+
+Allows arbitary python to be run against the app instance
```
Comment (conbon) [inline services/events/project/scripts/README.md]: "arbitrary"

File: services/events/project/scripts/event_tool.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from project import db
+from project.api.models import Event
+from datetime import datetime
+
+
+def soft_delete_events(source):
+    now = datetime.now().isoformat()
```
Comment (conbon) [inline services/events/project/scripts/event_tool.py]: Doesn't really make too much of a difference I can see right now but should this not be inside the loop so the timestamps are more accurate and not all identical when marking them 'deleted'?

File: services/events/project/scripts/README.md (hunk 1 above)
Comment (apoclyps) [reply services/events/project/scripts/README.md]:
```suggestion
### Running one off tasks
```

File: services/events/project/scripts/README.md (hunk 2 above)
Comment (apoclyps) [reply services/events/project/scripts/README.md]:
```suggestion
Allows arbitrary python to be run against the app instance
```

File: services/events/project/scripts/event_tool.py (hunk 3 above)
Comment (apoclyps) [reply services/events/project/scripts/event_tool.py]: Good call; The soft deleted `deleted_at` was intended to be identical to show they were updated in the same batch but the `updated_at` field should contain the time it was actually updated. Will update

**Comment by apoclyps** [comment]
@conbon cheers for the review; feedback addressed if you're happy to approve. I've sent you an invite to join as a [collaborator](services/events/project/scripts/README.md) 👍

**Pull request merged**